vsutil/__init__.py

```diff
@@ -275,3 +280,13 @@ def _resolve_enum(enum: Type[E], value: Any, var_name: str, module: Optional[str
         return enum(value)
     except ValueError:
         raise ValueError(f'{var_name} must be in {_readable_enums(enum, module)}.') from None
+
+
+def _disallow_variable(clip: vs.VideoNode, /) -> None:
```

Comment:
What about this signature:
```py
def _disallow_variable(clip: vs.VideoNode, *, format: bool = True, resolution=True) -> None
```

It would be more reusable.